.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 2018,
```

Comment:
Unfortunate that we need to specify the ecma version here. The latest version should be the default, rather than ES5. Prefer setting it to the latest year, which from the eslint docs sounds like it's 2020.